### DISCUSSION TITLE
Neo Geo audio (music and sound effects) cuts out after a few minutes of play.

### DISCUSSION THREAD

**Discussion by ljewe7** [OnionUI/Onion#984] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

Been like this on several versions of Onion now.

### Miyoo Firmware Version

Mini: 202204190828, Mini Plus on the latest (for WIFI to work)

### Provide a clear and concise description of the issue

 Neo Turf Masters is the game I've seen to happen on most. Sometimes it comes back slowly and and in parts. Sometimes the audio sounds corrupt and is even worst - pain loud crackling. 

### Steps to Reproduce

Just play for a bit

### Provide any additional context or information that may help us investigate your issue

Nothing I can think of

**Comment by Aemiii91** [comment]
Played 10 minutes of Turf Masters, sound worked fine. Have you added `neogeo.zip` in both `BIOS` and `Roms/NEOGEO` folders?
